claims_hosp/delphi_claims_hosp/config.py

```diff
@@ -3,7 +3,7 @@
 
 Author: Maria Jahja
 Created: 2020-06-01
-Modified: 2020-09-27
+Modified: 2021-12-11
```

Comment:
is there a reason we're not relying on git to track this?

Reply:
Possible reason: production deployments are stripped of all .git metadata, so markup like this would be the only way to verify which changes are in effect. It's not a _good_ reason (perennially backburnered is having deployment add a VERSION file with the gitref, which would be better) but it's better than nothing